New Merc shuttle and base

juxjux9930 opened the pull request:
🆑 
maptweak: The Desperado has been replaced with the Cyclopes; the Cyclopes is larger, faster, and better stocked than the Desperado.
maptweak: The Tersten Tenacity, the old merc spawn, has been replaced with a streamlined asteroid base. 
maptweak: The D4 fore airlock is now a proper docking port that the Cyclopes can use.
/ 🆑 

Let's talk about how this is an improvement, yeah?

One! The Cyclopes performs better as a ship, being faster, better fueled, and better stocked than the desperado!
Two! The new map is smaller and streamlined so mercs have to do as little at-base prep as possible before headed to the Torch!
Three! The Cyclopes can dock! The desperado can't _really_ dock!
Four! The way the cyclopes and it's spawn are flavored up, in addition to the chameleon outfits and the paper+pen, expands player choice on who exactly the merc team is that round!
Five! There's some tutorial notes laying around for concepts not all players might be familiar with, like how to not use all your fuel in one click, how to set a hardsuit up, and how to cycle a voidsuit!

Willing to elaborate on each of these and more in the appropriate channels on discord.

juxtaposed9930 commented:
First update batch. Adds some toolbelts, switches the interior airlock thingamajig to a button instead of a controller, and adds Flashing Barrels, a sterile diona nymph that I thought looked cute in a merc hardsuit helmet.

SierraKomodo reviewed (dismissed):
- Cyclopes docking controller seems to get stuck on 'Docking' when the map spawns.

- You have a light and a charger on the same wall where the sprites overlap near the helm and in the SW room, making the chargers hard to click.
![dreamseeker_xnxTmXzk9W](https://user-images.githubusercontent.com/11140088/98979733-8ea66880-24d0-11eb-86b3-c9a2774e06c6.png)

- Having a hailing intercom next to the helm would be consistant with the changes I've made to other shuttles.
- Your nymph that has a description of wearing a tiny helmet doesn't have a tiny helmet.
- The nanomed plus is a dense object - Pixel shifting it to another tile is probably not the best method because you can't walk on the tile it was placed on as a result. You /can/ get around this by changing the `density` var to `0` on the mapped object, but it'd be more consistant to just map it in palce of the wall like the one in medbay.

- Runtimes that occur when loading the merc map template
```
[09:01:28] Runtime in code/game/objects/structures/catwalk.dm,16: Deleting duplicate of The catwalk (/obj/structure/catwalk) ([0x202f4a5]) (the plating) (75,88,18) (/turf/simulated/floor/plating)
  proc name: crash at (/proc/crash_at)
  usr: Zhai Kovac (/mob/observer/ghost) ([0x3000128]) (the deck) (95,103,3) (/turf/simulated/floor/tiled/monotile) (sierrakomodo)
  usr.loc: The deck (/turf/simulated/floor/tiled/monotile) ([0x101888e]) (95,103,3) (/area/hallway/primary/thirddeck/center)
  src: null
  call stack:
  crash at("Deleting duplicate of The catw...", "code/game/objects/structures/c...", 16)
  the catwalk (/obj/structure/catwalk): Initialize(1)
  Atoms (/datum/controller/subsystem/atoms): InitAtom(the catwalk (/obj/structure/catwalk), /list (/list))
  Atoms (/datum/controller/subsystem/atoms): InitializeAtoms()
  Mercenary Base (/datum/map_template/ruin/antag_spawn/mercenary): init atoms(/list (/list))
  Mercenary Base (/datum/map_template/ruin/antag_spawn/mercenary): load new z(1)
  /datum/antagonist/mercenary (/datum/antagonist/mercenary): add antagonist(/datum/mind (/datum/mind), 1, 1, 0, 1, 1)
  /datum/mind (/datum/mind): Topic("src=\[0x21049fae];add_antagoni...", /list (/list))
  SierraKomodo (/client): Topic("src=\[0x21049fae];add_antagoni...", /list (/list), /datum/mind (/datum/mind))

[09:01:29] Runtime in lighting_overlay.dm,69: cannot read from list
  proc name: update overlay (/atom/movable/lighting_overlay/proc/update_overlay)
  src:  (/atom/movable/lighting_overlay)
  src.loc: the deck (76,95,18) (/turf/simulated/floor/tiled/techfloor)
  call stack:
   (/atom/movable/lighting_overlay): update overlay()
   (/atom/movable/lighting_overlay): New(the deck (76,95,18) (/turf/simulated/floor/tiled/techfloor), 0)
  the deck (76,95,18) (/turf/simulated/floor/tiled/techfloor): get lumcount(0, 1)
  Flashing Barrels (/mob/living/carbon/alien/diona/sterile): handle environment(/datum/gas_mixture (/datum/gas_mixture))
  Flashing Barrels (/mob/living/carbon/alien/diona/sterile): Life(20, 44, Mobs (/datum/controller/subsystem/processing/mobs))
  Flashing Barrels (/mob/living/carbon/alien/diona/sterile): Life(20, 44, Mobs (/datum/controller/subsystem/processing/mobs))
  Flashing Barrels (/mob/living/carbon/alien/diona/sterile): Life(20, 44, Mobs (/datum/controller/subsystem/processing/mobs))
  Mobs (/datum/controller/subsystem/processing/mobs): fire(0)
  Mobs (/datum/controller/subsystem/processing/mobs): ignite(0)
  Master (/datum/controller/master): RunQueue()
  Master (/datum/controller/master): Loop()
  Master (/datum/controller/master): StartProcessing(0)

[09:01:44] Runtime in buildandrepair.dm,12: Cannot read /decl/machine_construction/fra... (/decl/machine_construction/frame/awaiting_circuit).type
  proc name: on update icon (/obj/machinery/constructable_frame/computerframe/on_update_icon)
  src: the computer frame (/obj/machinery/constructable_frame/computerframe/deconstruct)
  src.loc: the deck (68,86,18) (/turf/simulated/floor/tiled/techfloor)
  call stack:
  the computer frame (/obj/machinery/constructable_frame/computerframe/deconstruct): on update icon()
  the computer frame (/obj/machinery/constructable_frame/computerframe/deconstruct): update icon()
  the computer frame (/obj/machinery/constructable_frame/computerframe/deconstruct): update icon()
  Icon Updates (/datum/controller/subsystem/processing/icon_update): fire(0, 0)
  Icon Updates (/datum/controller/subsystem/processing/icon_update): ignite(0)
  Master (/datum/controller/master): RunQueue()
  Master (/datum/controller/master): Loop()
  Master (/datum/controller/master): StartProcessing(0)
```

juxjux9930 committed:
Out with the Desperado, in with the.. chunky desperado


Various things

Renames desperado preemptively. Allows for proper docking if codes are given. More work on the shuttle itself. Makes shuttle not consume horrible amounts of fuel.
Desperado no longer requires mercs to devote skillpoints to fly


Some tweaks, begins new merc shuttle 


Cyclopes, maybe. Naming is hard.


Hammerhead is a better name.


Adds breathable asteroid turf for my own sanity


Does it REALLY need two areas


Hydrogen is a bane on mercs everywhere.


Adds flavorful tutorial notes for the shuttle & suit cycler


Hardsuit tutorial note 


Fixes tutorial typos. Removes vestige tenacity - almost functional, but not pretty


Adds kelotane and dexalin bottles. Some mapping.

Makes my life easier.
Abandoned mine, more like abandoned gone


Merc base, new. 


Switches merc base to a funky reactor for testing


The shuttle doesn't work. Trying to make the shuttle work.


Fixes the shuttle.


Uses the correct appearance mirror.


Merc base is never known


Gives vox *some* love. No throwers though.


Minor QoL + merc nymph


More little fixes

juxtaposed9930 commented:
If the nymph is being difficult I'll just delete it, it's not worth the hassle. I'll get on the other stuff right away.

juxtaposed9930 commented:
Fiddled with some of the airlock's tags and haven't gotten any of the docking issue since. Deleting the nymph got rid of that runtime - as for the console, I'm not sure why this is appearing here if it didn't appear with the old desperado. I've tried replacing the `deconstruct` object that desperado uses for the empty teleporter computer with a normal computer frame - might fix that?

juxtaposed9930 commented:
Fixed some lockers with the wrong access.

juxtaposed9930 commented:
Rebased to bring up to dev.

juxtaposed9930 commented:
Moved some poorly positioned lights and gave the cryo/spawn another door out.

juxtaposed9930 commented:
Closing while I take a bay break.
